PolicyServerElasticsearchContainer/pom.xml

```diff
@@ -444,6 +395,14 @@
                                     <basedir>/</basedir>
                                     <mode>tar</mode>
                                     <inline>
+                                        <fileSets>
+                                            <fileSet>
+                                                <fileMode>0755</fileMode>
+                                                <lineEnding>unix</lineEnding>
+                                                <directory>src/main/scripts/</directory>
```

Comment:
I'd like to change this to `src/main/scripts/startup.d/`.

Reply:
Moved the script to `src/main/startup.d/scripts`